src/main/resources/view/PersonListCard.fxml

```diff
@@ -22,49 +22,52 @@
+        <Button fx:id="share" onAction="#handleButtonAction" text="Share" HBox.hgrow="NEVER" minWidth="75"/>
       </HBox>
       <VBox spacing="5">
-        <FlowPane fx:id="tags" />
-        <FlowPane fx:id="dates" />
+        <padding>
+          <Insets bottom="0" left="0" right="0" top="5" />
+        </padding>
+        <FlowPane fx:id="tags" prefWrapLength="200" />
+        <FlowPane fx:id="dates" prefWrapLength="200" />
       </VBox>
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="remark" styleClass="cell_small_label" text="\$remark" />
-      <HBox fx:id="handles" spacing="8.0">
+      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" wrapText="true"/>
+      <Label fx:id="remark" styleClass="cell_small_label" text="\$remark" wrapText="true"/>
+
+      <FlowPane fx:id="handles">
         <HBox fx:id="facebook" alignment="CENTER_LEFT" styleClass="handle" spacing="4">
           <ImageView fx:id="facebookIcon" fitHeight="14" fitWidth="14" pickOnBounds="true" preserveRatio="true" />
-          <Label fx:id="facebookHandle" styleClass="cell_small_label" text="\$facebook" />
+          <Label fx:id="facebookHandle" styleClass="cell_small_label" text="\$facebook" maxWidth="200" />
         </HBox>
         <HBox fx:id="instagram" alignment="CENTER_LEFT" styleClass="handle" spacing="4">
           <ImageView fx:id="instagramIcon" fitHeight="14" fitWidth="14" pickOnBounds="true" preserveRatio="true" />
-          <Label fx:id="instagramHandle" styleClass="cell_small_label" text="\$instagram" />
+          <Label fx:id="instagramHandle" styleClass="cell_small_label" text="\$instagram" maxWidth="200" />
         </HBox>
         <HBox fx:id="telegram" alignment="CENTER_LEFT" styleClass="handle" spacing="4">
           <ImageView fx:id="telegramIcon" fitHeight="14" fitWidth="14" pickOnBounds="true" preserveRatio="true" />
-          <Label fx:id="telegramHandle" styleClass="cell_small_label" text="\$telegram" />
+          <Label fx:id="telegramHandle" styleClass="cell_small_label" text="\$telegram" maxWidth="200" />
         </HBox>
         <HBox fx:id="tiktok" alignment="CENTER_LEFT" styleClass="handle" spacing="4">
           <ImageView fx:id="tiktokIcon" fitHeight="14" fitWidth="14" pickOnBounds="true" preserveRatio="true" />
-          <Label fx:id="tiktokHandle" styleClass="cell_small_label" text="\$tiktok" />
+          <Label fx:id="tiktokHandle" styleClass="cell_small_label" text="\$tiktok" maxWidth="200" />
         </HBox>
         <HBox fx:id="twitter" alignment="CENTER_LEFT" styleClass="handle" spacing="4">
           <ImageView fx:id="twitterIcon" fitHeight="14" fitWidth="14" pickOnBounds="true" preserveRatio="true" />
-          <Label fx:id="twitterHandle" styleClass="cell_small_label" text="\$twitter" />
+          <Label fx:id="twitterHandle" styleClass="cell_small_label" text="\$twitter" maxWidth="150" />
```

Comment:
Any reason why `maxWidth` is 150 here, compared to 200 for the rest?

Reply:
Oops forgot to change this to 200... Will change it back soon! (but actually since twitter handles are <15 characters it will never reach LOL)

Reply:
Right, I forgot about that. I think it's probably easier to edit to 200 still for consistency with the rest, just in case.